gen3/docs/blog/authors.yml

```diff
@@ -1,10 +1,9 @@
 authors:
   qureshi:
     name: Jawad Qureshi
-    description: Lead Platform Engineer
+    description: Lead Platform Engineer, CTDS
     avatar: https://github.com/jawadqur.png
   elise:
     name: Elise Castle
-    description: Platform Engineer - Gen3
-    url: https://github.com/EliseCastle23
+    description: Platform Engineer, CTDS
```

Comment:
I am curious why you are changing this from Gen3 to CTDS. Since this is all Gen3 documentation, and since I think the users will be more familiar with Gen3 than with CTDS (especially since we are inconsistent about CTDS vs CDIS) -- it feels like it would be better to use Gen3 . We can discuss more -- doesn't need to hold up merging this PR.